vassal-app/src/main/java/VASSAL/counters/PlaceMarker.java

```diff
@@ -713,6 +731,10 @@ protected BooleanConfigurer createAboveConfig() {
       return null;
     }
 
+    protected BooleanConfigurer createCopyConfig() {
+      return null;
+    }
```

Comment:
^^^^ HERE ^^^^

Reply:
This is where an IDE helps you out. For me, I see:

![image](https://github.com/vassalengine/vassal/assets/1622667/1e1ad42a-c480-4b55-926f-484da0b8184c)

Which tells me that this method is overidden in some subclass. Clicking on the little symbol (or clicking on the method name and Ctrl-V) takes me to:

![image](https://github.com/vassalengine/vassal/assets/1622667/d9b42071-dd13-4a4e-a170-0708305a3506)

Personally, I would necessarily not have coded it this way, as the superclass needs too much knowledge of what subclasses might do, but Decorator Editors are a bitch to subclass.

Reply:
I see. This is an awkward design, but seems to have been the way it is since 2008; not much we can do about it now. Thanks for the explanation.